/bin/bash: /apps/compilers/anaconda3-2023.3/lib/libtinfo.so.6: no version information available (required by /bin/bash)
sh: 1: bedtools: not found
sh: 1: bedtools: not found
Traceback (most recent call last):
  File "/home/samarth/ctcf_sequence/fimo_analysis/fimo_neighbourhood_analysis.py", line 164, in <module>
    main()
  File "/home/samarth/ctcf_sequence/fimo_analysis/fimo_neighbourhood_analysis.py", line 63, in main
    fimo_to_neighbourhood(fimo_filepath, output_dir, genome_fasta, n_a, n_b)
  File "/home/samarth/ctcf_sequence/fimo_analysis/functions_script.py", line 111, in fimo_to_neighbourhood
    fimo_minus_seqs = list(SeqIO.parse(os.path.join(output_dir, 'fimo_minus.fasta'), 'fasta'))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samarth/.local/lib/python3.12/site-packages/Bio/SeqIO/__init__.py", line 625, in parse
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samarth/.local/lib/python3.12/site-packages/Bio/SeqIO/FastaIO.py", line 187, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
  File "/home/samarth/.local/lib/python3.12/site-packages/Bio/SeqIO/Interfaces.py", line 64, in __init__
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/samarth/ctcf_sequence_data/output/fimo/H1esc_CTCF_50_50/fimo_minus.fasta'
sh: 1: bedtools: not found
sh: 1: bedtools: not found
Traceback (most recent call last):
  File "/home/samarth/ctcf_sequence/fimo_analysis/fimo_neighbourhood_analysis.py", line 164, in <module>
    main()
  File "/home/samarth/ctcf_sequence/fimo_analysis/fimo_neighbourhood_analysis.py", line 63, in main
    fimo_to_neighbourhood(fimo_filepath, output_dir, genome_fasta, n_a, n_b)
  File "/home/samarth/ctcf_sequence/fimo_analysis/functions_script.py", line 111, in fimo_to_neighbourhood
    fimo_minus_seqs = list(SeqIO.parse(os.path.join(output_dir, 'fimo_minus.fasta'), 'fasta'))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samarth/.local/lib/python3.12/site-packages/Bio/SeqIO/__init__.py", line 625, in parse
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samarth/.local/lib/python3.12/site-packages/Bio/SeqIO/FastaIO.py", line 187, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
  File "/home/samarth/.local/lib/python3.12/site-packages/Bio/SeqIO/Interfaces.py", line 64, in __init__
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/samarth/ctcf_sequence_data/output/fimo/HFF_CTCF_50_50/fimo_minus.fasta'
sh: 1: bedtools: not found
sh: 1: bedtools: not found
Traceback (most recent call last):
  File "/home/samarth/ctcf_sequence/fimo_analysis/fimo_neighbourhood_analysis.py", line 164, in <module>
    main()
  File "/home/samarth/ctcf_sequence/fimo_analysis/fimo_neighbourhood_analysis.py", line 63, in main
    fimo_to_neighbourhood(fimo_filepath, output_dir, genome_fasta, n_a, n_b)
  File "/home/samarth/ctcf_sequence/fimo_analysis/functions_script.py", line 111, in fimo_to_neighbourhood
    fimo_minus_seqs = list(SeqIO.parse(os.path.join(output_dir, 'fimo_minus.fasta'), 'fasta'))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samarth/.local/lib/python3.12/site-packages/Bio/SeqIO/__init__.py", line 625, in parse
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samarth/.local/lib/python3.12/site-packages/Bio/SeqIO/FastaIO.py", line 187, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
  File "/home/samarth/.local/lib/python3.12/site-packages/Bio/SeqIO/Interfaces.py", line 64, in __init__
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/samarth/ctcf_sequence_data/output/fimo/H1esc_CTCF_0_0/fimo_minus.fasta'
sh: 1: bedtools: not found
sh: 1: bedtools: not found
Traceback (most recent call last):
  File "/home/samarth/ctcf_sequence/fimo_analysis/fimo_neighbourhood_analysis.py", line 164, in <module>
    main()
  File "/home/samarth/ctcf_sequence/fimo_analysis/fimo_neighbourhood_analysis.py", line 63, in main
    fimo_to_neighbourhood(fimo_filepath, output_dir, genome_fasta, n_a, n_b)
  File "/home/samarth/ctcf_sequence/fimo_analysis/functions_script.py", line 111, in fimo_to_neighbourhood
    fimo_minus_seqs = list(SeqIO.parse(os.path.join(output_dir, 'fimo_minus.fasta'), 'fasta'))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samarth/.local/lib/python3.12/site-packages/Bio/SeqIO/__init__.py", line 625, in parse
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samarth/.local/lib/python3.12/site-packages/Bio/SeqIO/FastaIO.py", line 187, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
  File "/home/samarth/.local/lib/python3.12/site-packages/Bio/SeqIO/Interfaces.py", line 64, in __init__
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/samarth/ctcf_sequence_data/output/fimo/HFF_CTCF_0_0/fimo_minus.fasta'
